pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/dao/impl/AlternateCareFacilityDAOImpl.java

```diff
@@ -178,11 +190,21 @@ private AlternateCareFacilityEntity getEntityById(Long id) {
 		return entity;
 	}
 	
+	private List<AlternateCareFacilityEntity> getEntityByIdentifier(String identifier) {
+		Query query = entityManager.createQuery( "SELECT DISTINCT a "
+				+ "FROM AlternateCareFacilityEntity a "
+				+ "LEFT OUTER JOIN FETCH a.lines lines "
+				+ "where (a.identifier LIKE :identifier)"
+				+ "ORDER BY lines.order", AlternateCareFacilityEntity.class );
+		query.setParameter("identifier", identifier);
+		return query.getResultList();
+	}
+	
 	private List<AlternateCareFacilityEntity> getEntityByName(String name) {
 		Query query = entityManager.createQuery( "SELECT DISTINCT a "
 				+ "FROM AlternateCareFacilityEntity a "
 				+ "LEFT OUTER JOIN FETCH a.lines lines "
-				+ "where (a.name LIKE :name) "
+				+ "where (a.name LIKE :name)"
```

Comment:
No.. that's the method above this one. This method is for a lookup by friendly name, which is now called "name".